guizero/base.py

```diff
@@ -406,11 +406,20 @@ def _close_window(self):
     def update(self):
         self.tk.update()
 
+    def full_screen(self, keybind="<Escape>"):
```

Comment:
Should we catered for `None` being passed the keybind event, so that you can go full screen without any backdoor. Agree that the default should be Escape tho.